app/assets/stylesheets/top-bar.scss

```diff
@@ -2,6 +2,11 @@
 @import 'variables';
 @import 'mixins';
 
+:target {
+  padding-top: var(--header-height) !important;
```

Comment:
Is `!important` strictly necessary here? I'm not a fan of adding it when there are alternatives.

Reply:
`:target` will have lower specificity, so I thought this would be problematic but it turns out that the visual effect is more or less the same. 

![lower-spec](https://user-images.githubusercontent.com/60417145/83911313-0e6a2300-a73a-11ea-9fea-18a5674c1aea.png)

Here is what happens when I remove `!important`. 

![margin-override](https://user-images.githubusercontent.com/60417145/83911331-13c76d80-a73a-11ea-9033-c3e910c5dbba.png)

Yay still working 🙂